app/upgrades/v1_7/const.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	ratelimittypes "github.com/cosmos/ibc-apps/modules/rate-limiting/v10/types"
 
-	erc20types "github.com/cosmos/evm/x/erc20/types"
+	erc20types "github.com/xpladev/xpla/legacy/ethermint/x/erc20/types"
```

Comment:
I don't think we need to support backwards compatibility. Let's delete the v1_7 folder!